services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -1009,6 +1016,6 @@ export default compose(
     form: FORM.ADD_EDIT_INCIDENT,
     enableReinitialize: true,
     touchOnBlur: true,
-    touchOnChange: false,
+    touchOnChange: true,
```

Comment:
These forms were created with touchOnFalse: false and I have no idea why. The false setting created issues with validation messages showing up, and I did not notice any issues after I changed it.